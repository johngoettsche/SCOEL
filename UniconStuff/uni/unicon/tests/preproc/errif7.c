/* test error message: missing #endif with nested #ifs */

#if 1
#if 1
one
#endif
two
