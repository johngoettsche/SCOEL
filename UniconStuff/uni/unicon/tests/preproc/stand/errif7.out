#line 4 "errif7.c"

one

two
pp: File errif7.c; Line 3: no matching #endif for #if
